src/sentry/incidents/metric_alert_detector.py

```diff
@@ -133,11 +144,11 @@ def update(self, instance: Detector, validated_data: dict[str, Any]):
 
         instance.save()
 
-        create_audit_entry(
-            request=self.context["request"],
-            organization=self.context["organization"],
-            target_object=instance.id,
-            event=audit_log.get_event_id("DETECTOR_EDIT"),
-            data=instance.get_audit_log_data(),
-        )
+        # create_audit_entry(
```

Comment:
commented this out because it's no longer always being used with a request, hadn't dealt with it yet for the dual write case